src/main/java/com/zendesk/rollbar/RollbarLog4jAppender.java

```diff
@@ -27,27 +27,35 @@
 
 import static com.rollbar.api.payload.data.Level.CRITICAL;
 import static org.apache.log4j.Level.*;
+import java.util.*;
+import java.net.*;
 
 
 public class RollbarLog4jAppender extends AppenderSkeleton {
 
   private String accessToken;
   private String environment;
   private String url;
+  private String hostName;
   private com.rollbar.notifier.Rollbar client;
 
   @Override
   public synchronized void activateOptions() {
     super.activateOptions();
+    try {
+      InetAddress ip = InetAddress.getByName(hostName);
+    }catch(Exception e){
+      LogLog.error("Invalid hostName");
+    }
     if (this.accessToken != null && !this.accessToken.isEmpty() && this.environment != null
-        && !this.environment.isEmpty()) {
+            && !this.environment.isEmpty() && !this.hostName.isEmpty() && this.hostName != null) {
```

Comment:
hostname is not being used in this block - so remove the check

Reply:
Okay